src/customers/account-dashboard-approval-rules.md

```diff
@@ -19,6 +19,18 @@ _Company Approval Rules_
 
 Depending on the company role and permissions, users may have the option to edit, delete, or view approval rules.
 
+## Purchase order payment methods
+
+Purchase order approval supports both online and offline payment methods.
+
+All default offline payment methods are supported for purchase order approvals.
+For online payments, the following methods are supported:
+
+* Paypal Express
+* Braintree payments
+
+Note that using gift cards, store credit, or reward points with online payment methods for purchase orders is not supported. Enabling these features with online payments may cause some unexpected behavior. It is recommended to disable gift cards, store credit, and reward points when online payments are enabled for purchase orders.
+
```

Comment:
should also indicate that uusers can bypass POs w/ paypal if the merchant has paypal enabled to show in the minicart, product details page, or somewhere other than the checkout payment page

also include that changes to discounts, catalog price rules, shipping, etc will not propagate to a PO after creation but before payment

Reply:
We have a KB for that.